Code/Tools/AssetProcessor/native/ui/MainWindow.ui

```diff
@@ -1316,6 +1316,65 @@
             </property>
            </spacer>
           </item>
+          <item>
```

Comment:
The easiest way to see this .ui code diff is to use the hide the white space option